examples/workers/basic-module/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'worktop';
+import { listen } from 'worktop/modules';
+
+// binding: ENV variable
+declare const FALLBACK: string;
+
+const API = new Router();
+
+API.add('GET', '/greet/:name?', (req, res) => {
+	res.end(`Hello via Module, ${req.params.name || FALLBACK}!`);
```

Comment:
Even though this is a Module Worker, still uses/relies on `globalThis` binding availability.